pennylane/tape/operation.py

```diff
@@ -63,6 +63,8 @@ def operation_expand(self):
         operations decomposition, or if not implemented, simply
         the operation itself.
     """
+    # TODO: rename the following usage of JacobianTape to better describe that
+    # it's used for operation expansion
```

Comment:
Do you mean simply renaming it to e.g. `op_expansion_tape`? I'm not against it per se, but I'm thinking it's quite clear already from the method + docstring, and it's only restricted to this quite small method. Maybe this is just something that can be done in this PR (since it's a very small, cosmetic change only) in that case instead of leaving it as a future TODO? :slightly_smiling_face:

Reply:
The comment more so implies that we are creating a `JacobianTape` which is not necessarily used for jacobian computations. Perhaps we could a) rename  `JacobianTape` or b) create a new type of tape that is more specific to tape expansions.

Reply:
Oh, I see. In that case, this is a bit more complex. I though you meant renaming the name `tape` and not the tape `Jacobiantape`. :laughing:

Reply:
Oh good catch @antalszava! I just had a look at the source code for `QuantumTape.expand()`, and it looks like this doesn't need to be a `JacobianTape`, you should be able to replace it with a simple `QuantumTape` with no side effect.